src/bsp/platform/stm32f4/xi_bsp_rng_stm32f4.c

```diff
@@ -5,22 +5,43 @@
  */
 
 #include <xi_bsp_rng.h>
+#include <xi_debug.h>
```

Comment:
usually BSP modules don't depend on libxively. The xi_debug.h is not on the interface of the library this is considered as an internal tool. The meaning behind is the Xively C Client does not offer logging mechanism. But no further arguments agains using internal logging here. Especially of others can accept this @olgierdh @DellaBitta ?